scripts/check-siblings.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs');
+const path = require('path');
+const { env } = require('process');
+
+const updateArgument = '--update-sentry-capacitor';
+
+// Filters all Sentry packages but Capacitor, CLI and Wizard.
+const jsonFilter = /\s*\"\@sentry\/(?!capacitor|wizard|cli|typescript)(?<packageName>[a-zA-Z]+)\"\:\s*\"(?<version>.+)\"/;
+
+/**
+ * If user requested to ignore the post-install
+ * @return {Boolean} true if requested to skip the post-install check, false otherwise.
+ */
+function SkipPostInstall() {
+  if (env.npm_config_update_sentry_capacitor) {
+    // NPM.
+    return true;
+  }
+  else if (env.npm_config_argv && env.npm_config_argv.includes(updateArgument)) {
+    // YARN.
+    return true;
+  }
+  return false;
+}
+
+/**
+ * Gets the required sibling version for Sentry Capacitor.
+ * @return {String | undefined} The sibling version, undefined if not found.
+ */
+function GetRequiredSiblingVersion() {
+  if (env.npm_package_dependencies__sentry_browser) {
+    // Yarn.
+    return env.npm_package_dependencies__sentry_browser;
+  }
+
+  let capacitorPackagePath = '';
+  if (env.npm_package_json) {
+    // NPM.
+    capacitorPackagePath = env.npm_package_json;
+  }
+  else {
+    capacitorPackagePath = __dirname + '../Package.json';
```

Comment:
I think here should be `'../package.json'`  (lowercase) as here: https://github.com/getsentry/sentry-capacitor/blob/0.11.0/package.json

Reply:
Thanks for spotting this! it should be fixed on the version!